app/javascript/stylesheets/palette.scss

```diff
@@ -1,4 +1,5 @@
 $alabaster: #f2f1eb;
+$ibis-white: #f3e7e7; // light pinkish hue used in what's changing box
```

Comment:
This color looks close to what we're using for the background of the currently checked work type in the work type modal. I believe that is set to `rgba($primary, 0.1)`: https://github.com/sul-dlss/happy-heron/blob/main/app/javascript/stylesheets/toggle.scss#L7 I wonder if we should standardize on one or the other or if we're OK with this varying. Cc: @jcoyne

Reply:
@aaron-collier Yeah, don't know how much I care about this so :man_shrugging: